subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c

```diff
@@ -664,6 +664,14 @@ psa_status_t cracen_platform_keys_provision(const psa_key_attributes_t *attribut
 
 	NRF_MRAMC_Type *mramc = (NRF_MRAMC_Type *)DT_REG_ADDR(DT_NODELABEL(mramc));
 	nrf_mramc_config_t mramc_config, mramc_config_write_enabled;
+	nrf_mramc_readynext_timeout_t readynext_timeout, short_readynext_timeout;
+
+	nrf_mramc_readynext_timeout_get(mramc, &readynext_timeout);
+
+	/* Ensure all MRAMC words are committed by setting MRAMC READYNEXT timeout to 0 */
+	short_readynext_timeout.value = 0;
+	short_readynext_timeout.direct_write = true;
+	nrf_mramc_readynext_timeout_set(mramc, &short_readynext_timeout);
```

Comment:
I'm pretty sure this will wear out the MRAM like crazy. How about setting this only for the last byte/uint32 written?

Reply:
The platform keys are supposed to be written only once, so I don't think it'll be a problem.
However, I could move the nonce to be written last, since it's only 4 bytes, and encapsulate only the nonce with the READYNEXT timeout to 0. That should minimize the write cycles.